src/legacy/core_plugins/kibana/public/visualize/editor/_editor.scss

```diff
@@ -1,11 +1,7 @@
 .visEditor {
   @include flex-parent();
 
-  @include euiBreakpoint('l', 'xl') {
-    position: absolute;
-    width: 100%;
-    height: 100%;
-  }
+  height: 100%;
```

Comment:
When exaclty was the regression introduced?

Reply:
Hi @majagrubic !
I've found the root cause and updated the PR description